modules/creation/tutorial.md

```diff
@@ -245,49 +245,12 @@ As you can see, our three blocks of code (`__construct()`, `install()` and `unin
 
 This is a PrestaShop-specific object that allows to easily manage all the shop's settings. It stores its data on the `ps_configuration` database table.
```

Comment:
```suggestion
This PrestaShop-specific object allows you to easily manage all the shop's settings. It stores its data on the `PREFIX_configuration` database table.
```

I think we can try to use `PREFIX_tablename` as it is recommended to use the custom prefix.